lua/telescope/config.lua

```diff
@@ -101,6 +101,8 @@ function config.set_defaults(defaults)
   set("border", {})
   set("borderchars", { '─', '│', '─', '│', '╭', '╮', '╯', '╰'})
 
+  set("path_display", {})
+
```

Comment:
Can we get some documentation for `path_display` similar to `entry_prefix`? (Only for `path_display`, we have another pr that adds documentation for some of the other config options)

Reply:
Not sure I understand your question, but we do have https://github.com/nvim-telescope/telescope.nvim/wiki/Path-Display-Configuration. This PR updates the main README to the wiki.

Reply:
Okay first of all i wanna get this merged today because neovim 0.5 comes out tomorrow and this is a breaking backwards compatibility pr.

Yes we have documentation in the wiki and in the readme but some people dont always have access to that and vim has a great builtin tool for documentation. We are generating vim documentation from code. This is a occasion something like that:

```lua
set("path_display", {}, [[
  This documentation is autogenerated and will end up in the vim documentation!!!

  So we should do at least say here what this config field does and what the
  available options are. Similar to the other fields like sorting_strategy, 
  selection_strategy, ....
]])
```

If you don't have the time to finish it today, also because there will be merge conflicts after https://github.com/nvim-telescope/telescope.nvim/pull/922 is merged, I can finish some of this stuff i this merge request, so i am not stealing you contribution anyway thanks :)

Reply:
I think its best if the maintainers finish this part. I can't guarantee when I can get to this. And this will avoid more back-and-forth in the PR.

Reply:
I went ahead and added this kind of documentation, and fixed a bunch of merge conflicts.